src/sentry/tsdb/snuba.py

```diff
@@ -478,8 +448,12 @@ def __get_data_snql(
             query_result = raw_snql_query(
                 snql_request, referrer=f"tsdb-modelid:{model.value}", use_cache=use_cache
             )
+            if manual_group_on_time:
+                translated_results = {"data": query_result["data"]}
+            else:
+                translated_results = {"data": [reverse(d) for d in query_result["data"]]}
```

Comment:
Why do we need to reverse here now?

Reply:
As far as I could tell, its to handle `time` column parsing in this bit:
https://github.com/getsentry/sentry/blob/964343bac12fd57f82ff6b109ba91987a1d48db9/src/sentry/utils/snuba.py#L1384-L1389

`search_issues` didn't need to be reversed since we baked the `to_timestamp` logic into the select clause here:
https://github.com/getsentry/sentry/blob/7304f42b4f668e4e946d0470f3a160ec27cc4a00/src/sentry/issues/query.py#L21